types/ckeditor__ckeditor5-essentials/tslint.json

```
@@ -0,0 +1,14 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "ban-types": false,
+        "no-empty-interface": false,
+        "prefer-switch": false,
+        "quotemark": [
+            true,
+            "double",
+            "avoid-escape",
+            "avoid-template"
+        ]
+    }
```

```suggestion
    "extends": "dtslint/dt.json"
```
why you're adding those? it passes without exclusions,
```bash
Testing 1 changed packages: ckeditor__ckeditor5-essentials
Testing 1 dependent packages: ckeditor__ckeditor5-essentials
Installing NPM dependencies...
1> ckeditor__ckeditor5-essentials START
2> ckeditor__ckeditor5-essentials START
2> ckeditor__ckeditor5-essentials OK
1> ckeditor__ckeditor5-essentials OK
```